src/components/registration/steps.jsx

```diff
@@ -768,11 +773,11 @@ class OrganizationStep extends React.Component {
                 id: `teacherRegistration.${choice}`
             })
         }));
-        
+
```

Comment:
Remove the unnecessary blank lines added.

Reply:
I'm going to blame Atom for this one. 😝